lib/config.go

```diff
@@ -195,6 +228,132 @@ func PrintSampleConfig() {
 
 # Generate Liveness Agent Data
 # liveness_agent = true
+
+
+# Matrix settings for Compliance Generation
+[matrix]
+
+  [matrix.samples]
```

Comment:
This looks like a very complete matrix, not a sample one. Could we make it a bit shorter? otherwise the config generator will create a very long "sample" config that might be overwhelming. 

An idea could be to generate another file with this data called `/path/to/compliance-matrix.toml` and then in the sample config add a line like you did with `compliance_sample_reports_dir `, for example:
```toml
# Explain explain explain what is this file for, muchas gracias!
# compliance_matrix = "/path/to/compliance-matrix.toml"
```